### PR TITLE
add mission and resource links

### PR DESCRIPTION
**Pull request by jehende**
Task: task-[work-item-number]

Link: preview-[pull-request-number]

[Explain the context of this pull request here]

## Testing

1. Step one
2. Step two
   Expected result:

## Additional information

[Optional]

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,7 +3,18 @@
...
-Welcome to the Atlas Design project! This repository holds the source code backing appearance of docs.microsoft.com and Microsoft Learn.
+Welcome to the Atlas Design project! This repository holds the source code backing the Atlas Design System. 
+
+## Our mission
+Atlas leads a consistent experience for our customers, improving quality assurance, and driving down development times across the DevRel web properties. This Design System serves designers, PMs, and developers across DevRel to build accessible, and high quality experiences at scale.
+
+
+## Process
+
+- [New patterns](https://dev.azure.com/azurecom/ACOM/_wiki/wikis/ACOM.wiki/3960/GitHub-Discussions-Process)
```
Comment (wibjorn) [inline README.md:14]: Question, should this live in this repo, rather than a private wiki?

File: README.md [hunk 2]
```diff
@@ -3,7 +3,18 @@
...
+- Design Contribution Guidelines
+- [Development Contribution Guidelines](https://dev.azure.com/azurecom/ACOM/_wiki/wikis/ACOM.wiki/3960/GitHub-Discussions-Process)
```
Comment (wibjorn) [inline README.md:16]: Should this be removed and added after the Eng guidelines have been merged?

File: README.md (hunk 1 above)
Comment (jehende) [reply README.md:14]: I didn't realize this was public facing! It's probably best to live on the Atlas site.

**Comment by jehende** [comment]
@wibjorn removed the links and updated the mission to support this idea of connected content

**Review by wibjorn — APPROVED** [review]

**Pull request merged**